components/infobox/wikis/starcraft2/infobox_map_custom.lua

```diff
@@ -0,0 +1,79 @@
+local Map = require('Module:Infobox/Map')
+local Template = require('Module:Template')
+local Variables = require('Module:Variables')
+local StarCraft2Map = {}
+
+function StarCraft2Map.run(frame)
+	Map.getNameDisplay = StarCraft2Map.getNameDisplay
+	Map.addCustomCells = StarCraft2Map.addCustomCells
+	Map.addToLpdb = StarCraft2Map.addToLpdb
+	return Map.run(frame)
```

Comment:
It would be nice if we kept `run` reserved for the invokes and use `createInfobox` here like we do in Infobox/League